main.js

```diff
@@ -122,6 +128,27 @@ app.whenReady().then(async () => {
     }
   });
 
+  ipc.on(
+    'gitify:update-keyboard-shortcut',
+    (_, { enabled, keyboardShortcut }) => {
+      console.log('enabled', enabled);
+
+      console.log('keyboardShortcut', keyboardShortcut);
```

Comment:
Should these still be here? :)

Reply:
not at all 🙈

Reply:
Removed 🧼 

Great catch @bmulholland  🙇